Sources/Widgets/Widgets3D/AngleWidget/behavior.js

```diff
@@ -5,7 +5,7 @@ const MAX_POINTS = 3;
 
 export default function widgetBehavior(publicAPI, model) {
   model.classHierarchy.push('vtkAngleWidgetProp');
-  let isDragging = null;
+  model._isDragging = null;
```

Comment:
Instead of typing this as `null`, I would suggest typing it as `boolean` and having a default assignment of `false`. The same goes with all other instances of `_isDragging` instantiation,